src/lib/holocene/table/paginated-table/api-paginated.svelte

```diff
@@ -0,0 +1,229 @@
+<script lang="ts">
```

Comment:
The rename is gonna cause a BUNCH of tables in cloud to need re-imported when we update the component. No notes just saying. Compilation errors will help there

Reply:
We're currently only using this table in one place which was updated [here](https://github.com/temporalio/ui/pull/2261/files#diff-9658934782fa9de1d3992e17315fd3516efec1cd7a6cc8da42aeebd19f9ca130R5), so imports have been covered ✔️